packages/fast-components-react-msft/src/pivot/pivot.props.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from "react";

```

Comment:
This file can just be a ts file, no need for it to be tsx unless there is actually jsx templating syntax, it theoretically makes testing slower as it needs to pass through two transpiling stages if you use .tsx vs .ts.